starter_code/src/movies.js

```diff
@@ -5,9 +5,57 @@
 // Get the average of all rates with 2 decimals 
 
 
+
+
+
+
+
+function ratesAverage (movies) {
+    var sumMo = movies.reduce(function(sum,item){
+        return sum + parseFloat(item.rate);
+    }, 0);
+    var average = Math.round(sumMo/movies.length*100)/100;
+    return average;
+}
+
+
 // Get the average of Drama Movies
 
 
+// var dramaFi = movies.filter(function (item) {
+//     if (item.genre === 'Drama') {
+//         return true;
+//     } else {
+//         return false;
+//     }   
+// } );
+
+// function dramaMoviesRate (){
+
+// }
+
+
+
+
+
+
+function dramaMoviesRate (movies){
+
+    var dramaFi = movies.filter(function(movies) { 
+        return movies.genre.includes('Drama');      
+  });
+    var sumMo = dramaFi.reduce(function(sum,item){
```

Comment:
Don't you feel like you are repeating yourselves? After filtering the array, you can reuse the function that you write before (ratesAverage) to calculate the average.